src/index.js

```diff
@@ -1 +1,134 @@
 // Your server code here...
+import express from 'express';
+import mongoose from 'mongoose';
+import ActionHistoryModel from './models/ActionHistoryModel';
+import bodyParser from 'body-parser';
+import ContactRouter from './routes/ContactRouter';
+
+const app = express();
+
+// Connect to our mongo database
+mongoose.Promise = global.Promise;
+mongoose.connect('mongodb://localhost/contact-list');
+
+const PORT = 3001;
+
+app.use(bodyParser.json());
+
+app.use((request, response, next) => {
+  response.header('Access-Control-Allow-Origin', '*');
+  response.header('Access-Control-Allow-Headers', 'Origin, X-Requested-With, Content-Type, Accept');
+  response.header('Access-Control-Allow-Methods', 'POST, GET, PUT, DELETE');
+  next();
+});
+
+app.use(ContactRouter);
+
+// app.get('/contacts', (request, response) => {
+//   ContactModel.find({}).exec()
+//     .then(data => {
+//       return response.json(data);
+//     })
+//     .catch(error => {
+//       console.log(`Something went wrong with the contacts list! ${error}`);
+//     });
+// });
```

Comment:
I would always clean up after you are don with refactoring to not clutter the code.